lib/worker.js

```diff
@@ -10,8 +10,12 @@ const { Server } = metarhia.metacom;
 const { Auth } = require('./auth.js');
 
 (async () => {
+  const vm = require('vm');
+  const sandbox = { PORT: parseInt(process.env.PORT || 0) };
```

Comment:
```suggestion
  const sandbox = { process };
```
Not a good place for parsing ports, we can just pass process into sandbox here.